src/renderer/marketplace/incorporation/detail/index.jsx

```diff
@@ -155,6 +155,9 @@ const styles = theme => ({
 		'& ul li': {
 			lineHeight: '1.4em',
 			marginBottom: '0.5em'
+		},
+		'& a': {
+			color: '#93B0C1'
```

Comment:
@andregoncalves maybe better: 
`color: theme.palette.secondary.main`
or `import {typography} from 'selfkey-ui"`
`color: typography`

https://github.com/SelfKeyFoundation/selfkey-ui/blob/master/src/colors.tsx#L3
https://github.com/SelfKeyFoundation/selfkey-ui/blob/master/src/theme/selfkey-dark-theme.tsx#L48

Reply:
Got it.